.circleci/config.yml

```diff
@@ -0,0 +1,46 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/golang:1.9
+    working_directory: /go/src/github.com/robustperception/pushprox
+    steps:
+      - checkout
+      - run:
+          name: Build proxy
+          command: |
+              cd /go/src/github.com/robustperception/pushprox/proxy
+              go get 
+              go build
+      - run:
+          name: Build client
+          command: |
+              cd /go/src/github.com/robustperception/pushprox/client
+              go get 
+              go build
+      - run:
+          name: Download and untar node_exporter
+          command: |
+              wget https://github.com/prometheus/node_exporter/releases/download/v0.15.1/node_exporter-0.15.1.linux-amd64.tar.gz
+              tar xzf node_exporter-0.15.1.linux-amd64.tar.gz
+      - run:
+          name: Download, untar, and configure Prometheus
+          command: |
+              wget https://github.com/prometheus/prometheus/releases/download/v2.0.0/prometheus-2.0.0.linux-amd64.tar.gz
+              tar xzf prometheus-2.0.0.linux-amd64.tar.gz
+              cd prometheus-2.0.0.linux-amd64
+              rm prometheus.yml
+              printf "global:\n  scrape_interval:     5s\n  evaluation_interval: 5s\n  scrape_timeout: 1s\nscrape_configs:\n  - job_name: pushprox\n    proxy_url: http://127.0.0.1:8080\n    static_configs:\n      - targets: ['$(hostname -f):9100']" >> prometheus.yml
```

Comment:
A heredoc  would be more readable:

cat > prometheus,yml << EOF
blah
blah
blah
EOF

I'd also use a 1s scrape interval, and remove the other global settings